.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: NPM Install
+        run: npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
[Q] Can we not set it to current node `lts` variant ?

Reply:
We can use LTS variant, will do the change.

Reply:
Done.